react-frontend/src/components/StyleComponents/card.js

```diff
@@ -29,16 +31,20 @@ export const CardClickable = styled(Link).attrs({
   }
 `;
 
+export const CardClickableBody = styled.div.attrs({
```

Comment:
Restyling these cards has now broken common components cards (please see deployment)

Reply:
Good catch, thank you. Please see https://github.com/bcgov/digital.gov.bc.ca/pull/955/commits/9766faed283b7dc67cff18be97f25796e7b344b1